File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/examples/simple/CubeMinus1.java
package net.fwitz.math.main.binary.complex.functions.examples.simple;

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class CubeMinus1 {
    public static void main(String[] args) {
        new ComplexFunctionPlot("z^3 - 1")
                .computeFn(z -> z.pow3().minus(1))
                .domainX(-1.5, 1.5)
                .domainY(-1.5, 1.5)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/examples/simple/CubeMinus1.kt
package net.fwitz.math.main.binary.complex.functions.examples.simple

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object CubeMinus1 {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("z^3 - 1")
        .computeFn { z -> z.pow3 - 1 }
        .domainX(-1.5, 1.5)
        .domainY(-1.5, 1.5)
        .render()
}